README.md

```diff
@@ -296,6 +291,22 @@ def foo(obj: object) -> None:
         ...
 ```
 
+### Why am I getting incompatible argument type mentioning `_StrPromise`?
+
+The lazy translation functions of Django (such as `gettext_lazy`) return a `Promise` instead of `str`. These two types [cannot be used interchangeably](https://github.com/typeddjango/django-stubs/pull/1139#issuecomment-1232167698). The return type of these functions was therefore [changed](https://github.com/typeddjango/django-stubs/pull/689) to reflect that.
+
+If you encounter this error in your own code, use `StrPromise` or `StrOrPromise` from `django-stubs-ext`.
+
+```python
+from django.utils.translation import gettext_lazy as _
+
+from django_stubs_ext import StrOrPromise
+
+errors: list[StrOrPromise] = []
+errors.add(_('Error message'))
+```
+
+If this is reported on Django code, please report an issue or open a pull request.
```

Comment:
Sometimes also the correct solution is to call `str(value)` on the `StrPromise`, for example when passing to a third party library that expects a `str` type value. Although this may result in the translation being evaluated too early.

Reply:
Thanks, I can add this. Are you aware of any documentation on when it might be sufficient to evaluate translations (too) early?

Reply:
This looks like the perfect piece of documentation to link to: https://docs.djangoproject.com/en/4.1/topics/i18n/translation/#working-with-lazy-translation-objects

Reply:
In particular:

> Using gettext_lazy() and ngettext_lazy() to mark strings in models and utility functions is a common operation. When you’re working with these objects elsewhere in your code, you should ensure that you don’t accidentally convert them to strings, because they should be converted as late as possible (so that the correct locale is in effect). This necessitates the use of the helper function described next.

Reply:
Thanks, that's great. I revised the text and removed the code snippet.